src/AsseticBundle/Configuration.php

```diff
@@ -463,4 +468,20 @@ public function getWriteIfChanged()
     {
         return $this->writeIfChanged;
     }
+
+    /**
+     * @param bool $flag
+     */
+    public function setMergeActionAndRouterConfig($flag)
+    {
+        $this->mergeActionAndRouterConfig = (bool) $flag;
+    }
+
+    /**
+     * @return bool
+     */
+    public function canMergeActionAndRouterConfig()
+    {
+        return $this->mergeActionAndRouterConfig;
+    }
```

Comment:
To prevent potential regression in future releases, I will ask you to test that default configuration is set to `false`.
Test class for this file can be found here:
https://github.com/widmogrod/zf2-assetic-module/blob/master/tests/AsseticBundleTest/Configuration.php